code/Items/Data/ItemData.cs

```diff
@@ -0,0 +1,19 @@
+﻿namespace ssl.Items.Data
+{
+    public abstract class ItemData
```

Comment:
Not abstract nececerly. Because some Items have a basic ItemData i think.
I think we have to let the game create an Item (= item without any specification) and an ItemData define the item